doc/develop/manifest/index.rst

```diff
@@ -0,0 +1,53 @@
+:orphan:
+
+.. _west_projects_index:
+
+West Projects index
+###################
+
+See :ref:`external-contributions` for more information about
+this contributing and review process for imported components.
+
+Active Projects/Modules
++++++++++++++++++++++++
+
+The projects below are enabled by default and will be downloaded when you
+call `west update`. Many of the projects or modules listed below are
+essential for building generic Zephyr application and include among others
+hardware support for many of the platforms available in Zephyr.
+
+To disable any of the active modules, for example a specific HAL, use the
+following commands::
+
+        west config manifest.project-filter -- -hal_FOO
+        west update
+
+.. manifest-projects-table::
+   :filter: active
+
+Inactive and Optional Projects/Modules
+++++++++++++++++++++++++++++++++++++++
+
+
+The projects below are optional and will not be downloaded when you
+call `west update`. You can any of the the projects or modules listed below
+and use it to write application code.
+
+To enable any of the modules blow, use the following commands::
+
+        west config manifest.project-filter -- +nanopb
+        west update
+
+.. manifest-projects-table::
+   :filter: inactive
+
+External Projects/Modules
+++++++++++++++++++++++++++
```

Comment:
extra +

Reply:
@gmarull sphinx is not catching it here so something does seem amiss?

Reply:
not sure why, it used to catch problems like this IIRC